### ISSUE TITLE
Update "Copy password" icon in "Password Generator" tab

### ISSUE THREAD

**Issue by JackH3103** [keepassxreboot/keepassxc#7246]
## Overview
The "Copy password" button currently uses a clipboard icon, which is generally associated with the paste command.
IMO it should be changed to the de facto icon of two overlapping documents.

![Screenshot_20211228_131457](https://user-images.githubusercontent.com/23507259/147569998-74c4dbcd-5355-4710-934c-b90392ac5877.png)

## Steps to Reproduce
1. Open a database
2. Click "Password Generator" on the toolbar

## Expected Behavior
A copy icon should be displayed (usually two overlapping files)

## Actual Behavior
A clipboard icon is displayed (typically associated with pasting)

## Context
KeePassXC - Version 2.6.6
Revision: 9c108b9
Distribution: Flatpak

Operating System: Linux openSUSE Tumbleweed 20211226
Desktop Env: KDE Plasma 5.23.4
Windowing System: X11

**Comment by droidmonkey** [comment]
I don't think the overlapping pages icon will convey that intent in this view. I am not in favor of changing the current icon. 

This is the icon you mention: https://materialdesignicons.com/icon/content-copy

**Comment by JackH3103** [comment]
Personally when I see a clipboard icon I immediately think of pasting. The first time I looked at that window I initially thought there wasn't even an option to copy the password to clipboard until I started to question why there would even be a paste button on a password generator.

**Comment by droidmonkey** [comment]
I'd be in favor of the current clipboard but with an inward pointing arrow added. The pages icon is just too plain for my liking.

**Comment by JackH3103** [comment]
I agree with the arrow suggestion. I can see how two pages would also be confusing in this context

**Comment by droidmonkey** [comment]
Either of these might be better:

https://pictogrammers.com/library/mdi/icon/clipboard-arrow-right-outline/

https://pictogrammers.com/library/mdi/icon/clipboard-file-outline/

I prefer the arrow to come from outside to inside the clipboard, wouldn't be hard to edit.

**Comment by jgrabn** [comment]
Big +1 on this issue. Clipboard is the wrong icon. See for yourself: https://www.google.com/search?q=copy+icon

**Comment by droidmonkey** [comment]
Yes, as discussed above, we know and it is a deliberate choice.
